ci-operator/step-registry/openshift-extended/test/supplementary/openshift-extended-test-supplementary-commands.sh

```diff
@@ -0,0 +1,450 @@
+    fi
+    echo "TEST_FILTERS_SUPPLEMENTARY: \"${hardcoded_filters};${TEST_FILTERS_SUPPLEMENTARY:-}\""
+    echo "FILTERS_ADDITIONAL_SUPPLEMENTARY: \"${FILTERS_ADDITIONAL_SUPPLEMENTARY:-}\""
+    test_filters="${hardcoded_filters};${TEST_FILTERS_SUPPLEMENTARY}"
+    if [[ -n "${FILTERS_ADDITIONAL_SUPPLEMENTARY:-}" ]]; then
+        echo "add FILTERS_ADDITIONAL_SUPPLEMENTARY into test_filters"
+        test_filters="${hardcoded_filters};${TEST_FILTERS_SUPPLEMENTARY};${FILTERS_ADDITIONAL_SUPPLEMENTARY}"
+    fi
+    echo "${test_filters}"
+    handle_filters "${test_filters}"
+    echo "------------------the case selected------------------"
+    selected_case_num=$(cat ./case_selected|wc -l)
+    if [ "W${selected_case_num}W" == "W0W" ]; then
+        echo "No Case Selected"
+        if [ "W${FORCE_SUCCESS_EXIT}W" == "WnoW" ]; then
+            echo "do not force success exit"
+            exit 1
+        fi
+        echo "force success exit"
+        exit 0
+    fi
+    echo ${selected_case_num}
+    cat ./case_selected
+    echo "-----------------------------------------------------"
+
+    ret_value=0
+    set -x
+    if [ "W${TEST_PROVIDER}W" == "WnoneW" ]; then
+        extended-platform-tests run --max-parallel-tests ${TEST_PARALLEL} \
+        -o "${ARTIFACT_DIR}/extended.log" \
+        --timeout "${TEST_TIMEOUT_SUPPLEMENTARY}m" --junit-dir="${ARTIFACT_DIR}/junit" -f ./case_selected || ret_value=$?
+    else
+        extended-platform-tests run --max-parallel-tests ${TEST_PARALLEL} \
+        --provider "${TEST_PROVIDER}" -o "${ARTIFACT_DIR}/extended.log" \
+        --timeout "${TEST_TIMEOUT_SUPPLEMENTARY}m" --junit-dir="${ARTIFACT_DIR}/junit" -f ./case_selected || ret_value=$?
+    fi
+    set +x
+    set +e
+    rm -fr ./case_selected
+    echo "try to handle result"
+    handle_result
+    echo "done to handle result"
+    if [ "W${ret_value}W" == "W0W" ]; then
+        echo "success"
+    else
+        echo "fail"
+    fi
+    # it ensure the the step after this step in test will be executed per https://docs.ci.openshift.org/docs/architecture/step-registry/#workflow
+    # please refer to the junit result for case result, not depends on step result.
+    if [ "W${FORCE_SUCCESS_EXIT}W" == "WnoW" ]; then
+        echo "force success exit"
+        exit 1
+    fi
+
+    # summarize test results
+    echo "Summarizing test result..."
+    mapfile -t test_suite_failures < <(grep -r -E 'testsuite.*failures="[1-9][0-9]*"' "${ARTIFACT_DIR}" | grep -o -E 'failures="[0-9]+"' | sed -E 's/failures="([0-9]+)"/\1/')
+    failures=0
+    for (( i=0; i<${#test_suite_failures[@]}; ++i ))
+    do
+        let failures+=${test_suite_failures[$i]}
+    done
+    if [ $((failures)) == 0 ]; then
+        echo "All tests have passed"
+    else
+        echo "${failures} failures in openshift-extended-test" | tee -a "${SHARED_DIR}/openshift-e2e-test-qe-report-openshift-extended-test-failures"
```

Comment:
The file `openshift-e2e-test-qe-report-openshift-extended-test-failures` is used to track the number of test failures  for ginkgo tests. Using the same file name will overwrite the number of failures in step `openshift-extended-test`

Reply:
@sunzhaohua2 @liangxia 
so, change it to `echo "${failures} failures in openshift-extended-test-supplementary" | tee -a "${SHARED_DIR}/openshift-e2e-test-qe-report-openshift-extended-test-failures"`?